README.md

```diff
@@ -95,6 +95,4 @@ General instructions on _how_ to contribute should be stated with a link to [CON
 
 ## Credits and references
 
-1. Projects that inspired you
-2. Related projects
-3. Books, papers, talks, or other sources that have meaningful impact or influence on this project
+[Reference Document](https://github.com/NOAA-OWP/ngen/blob/ZacharyWills-review-1/doc/references.md)
```

Comment:
This should be \[Reference Document](doc/references.md).  Github will translate the local reference to the currently viewed branch.